gridded_etl_tools/utils/attributes.py

```diff
@@ -181,6 +187,19 @@ def ensemble_numbers(self) -> list[int]:
         """To be overwritten by actual ensemble datasets"""
         return list(None)
 
+    @property
+    def hindcast_steps(self) -> list[int]:
+        """To be overwritten by actual hindcast datasets"""
+        return list(None)
+
+    @classmethod
+    def irregular_update_cadence(self) -> bool | tuple[np.timedelta64, np.timedelta64]:
```

Comment:
Is it necessary to return False or could this return None instead? It's more common to use Optional[..] than to either return a bool or a value.